.github/workflows/wptagent_test.yml

```diff
@@ -36,6 +36,9 @@ jobs:
         python3 -m pip install --upgrade pip
         pip3 install -r .github/workflows/requirements.txt
 
+    - name: Analysing the code with Black
+      run: black --include font --check .
```

Comment:
Apply only to font-related files to begin with. This is what I intend to touch in this PR.